Cargo.toml

```diff
@@ -1,12 +1,12 @@
 [package]
 name = "blosc-src"
-version = "0.3.6"
+version = "0.3.7"
 authors = ["Magnus Ulimoen <magnus@ulimoen.dev>"]
 edition = "2021"
 rust-version = "1.64"
 build = "build.rs"
 links = "blosc"
-license-file = "c-blosc/LICENSE.txt"
```

Comment:
It wouldn't hurt to keep this, demonstrating that the license follows that of the underlying C source, right? Is this file now still included in the published crate? I've heard some folks/packagers that want the file even if there's an SPDX license specifier.

Reply:
I have only removed the largest offenders to keep the crate size down. I could use a whitelist instead in this crate